src/pages/guides/pnp-no-modal-multichain.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Creating a Multichain Connection with Web3Auth and React PnP no-modal SDK
+image: "guides/banners/multi.png"
+description: Learn how to connect to different blockchain using Web3Auth.
+type: guide
+tags: [multichain, polkadot, evm, cosmos, no-modal]
+date: Febreaury 9, 2024
+author: Web3Auth Team
+order: 1
```

Comment:
Please change the order for this guide, not sure we want this as our first guide. Also, change the guide order based on the date, and have a featured boolean label to display some guides as required.